ecf/scripts/stats/cam/jevs_cam_nam_firewxnest_grid2obs_stats_vhr_master.ecf

```diff
@@ -59,8 +59,7 @@ export COMOUT=${COMevs}/${STEP}/${COMPONENT}
 export pid=${pid:-$$}
 export outid=${outid:-"LL$job"}
 export DATA=${DATA:-${DATAROOT}/${jobid:?}}
-export config=$HOMEevs/parm/evs_config/${COMPONENT}/config.evs.${COMPONENT}_${MODELNAME}.${envir}
-source $config
+export config=$HOMEevs/parm/evs_config/${COMPONENT}/config.evs.${COMPONENT}_${MODELNAME}.prod
```

Comment:
@PerryShafran-NOAA Does this "export config" line need to be in the .ecf script if it's also in scripts/stats/cam/exevs_cam_nam_firewxnest_grid2obs_stats.sh? If you can remove this line from the dev drivers, you can probably remove it from the .ecf scripts too, right? Thanks for checking this. 

CC @ShelleyMelchior-NOAA

Reply:
Done.